### DISCUSSION TITLE
아이템 판매 수량과 판매 젬이 일치하지 않는 문제로 문의 드립니다.

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#1020] [5. Q&A]
고객센터로 문의를 했는데 자동 응답 메일만 오고 답이 없어서, 다시 한 번 여기에 문의 드립니다.
아이템 판매 수량과 판매 젬이 일치하지 않습니다.
혹시 테스터가 구매한 내역도 판매 수량에 포함이 되거나, 아니면 구매 시도만 해봐도 구매한 내역에 포함되나요?
이런게 아니라면 문제 해결 방법이 있을까요?
알려주세요🙏

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈를 파악하기 위해서 월드 id를 알려주시기를 부탁드립니다.

**Reply by ghost** [reply]
월드 ID 알려드렸습니다!

**Reply by Zepeto-TechEvangelist** [reply]
넵 맞습니다, 고객센터 쪽에도 어제 오후에 잘 접수는 되었다고 합니다. (자동메일은 접수 확인 메일일거에요~)
확인 후 고객센터 통해서 답변 나갈 예정입니다.
감사합니다~

**Reply by ghost** [reply]
감사합니다😊
